### ISSUE TITLE
Exception raised: 'MaskedColumn' object has no attribute '_mask'

### ISSUE THREAD

**Issue by maddiecain** [andycasey/smhr#272]
An exception was encountered using version 7b5597a on Python 2.7.6 | 32-bit | (default, Sep 15 2014, 17:36:57) [GCC 4.2.1 (Apple Inc. build 5666) (dot 3)]:

````python
Traceback (most recent call last):

  File "/Users/madelyncain/smhr/smh/gui/stellar_parameters.py", line 691, in measure_abundances
    self.update_scatter_plots()

  File "/Users/madelyncain/smhr/smh/gui/stellar_parameters.py", line 625, in update_scatter_plots
    self.update_trend_lines()

  File "/Users/madelyncain/smhr/smh/gui/stellar_parameters.py", line 738, in update_trend_lines
    yerr_column=yerr_column)

  File "/Users/madelyncain/smhr/smh/utils.py", line 143, in equilibrium_state
    A = np.vstack((np.ones_like(x), x)).T

  File "/Users/madelyncain/Library/Enthought/Canopy_32bit/User/lib/python2.7/site-packages/numpy/ma/core.py", line 5688, in <lambda>
    T = property(fget=lambda self: self.transpose())

  File "/Users/madelyncain/Library/Enthought/Canopy_32bit/User/lib/python2.7/site-packages/numpy/ma/core.py", line 2557, in wrapped_method
    mask = self._mask

AttributeError: 'MaskedColumn' object has no attribute '_mask'
````

**Additional details on how this exception occurred**:

**Comment by maddiecain** [comment]
Happens when i hit measure abundances

**Comment by alexji** [comment]
This is fixed in the newest version of branch v0.2.

(Note that a bug in continuum normalization was fixed too, and this should make the continuum fits better by default; but if you refit it will be a different shape.)
